### DISCUSSION TITLE
Bypass frida detection on Android

### DISCUSSION THREAD

**Discussion by ghost** [frida/frida#2573] [Q&A]
I encountered an Android game that crashes immediately as soon as Frida attach to a process. Does anyone know how to bypass frida detection on Android?

**Comment by ghost** [comment]
There are on Github scripts to bypass some checks, but it depends how the detection works, it may or may not be possible to bypass via Frida itself

**Comment by lyf-is-coding** [comment]
you can try unload the lib that contains frida detection

**Reply by nuttapolkung** [reply]
how to do this ?
